scanner/src/components/Documentation/index.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+  import Question from "$assets/question.svg"
+</script>
+
+<a class="relative documentation-wrapper" href="/">
+  <img alt="question" src={Question} />
+  <p class="documentation-label">Documentation</p>
```

Comment:
```suggestion
  <span class="documentation-label">Documentation</span>
```
You shouldn't use `<p>` inside `<a>`, it's a paragraph.